lib/comment.rb

```diff
@@ -0,0 +1,25 @@
+require 'active_record'
+
+if ENV['ENVIRONMENT'] == 'test'
+  ActiveRecord::Base.establish_connection(
```

Comment:
Rather than being in all model files, the connection setup should probably be factored in a file that would be required by the app when initialised — you can also use the method `ActiveRecord::Base.connected?` to make sure this is only run once

Reply:
All done, now thank you. Again, this makes a lot of sense